hamilton/function_modifiers/adapters.py

```diff
@@ -119,7 +119,25 @@ def __init__(
         self.kwargs = kwargs
         self.inject = inject_
 
-    def generate_nodes(self, fn: Callable, config: Dict[str, Any]) -> List[node.Node]:
+    def _select_param_to_inject(self, params: Dict[str, Type[Type]], fn: Callable) -> str:
+        if self.inject is None:
+            if len(params) == 1:
+                return list(params.keys())[0]
+            raise InvalidDecoratorException(
+                f"If the nodes produced by {fn.__qualname__} require multiple inputs, "
+                f"you must pass `inject_` to the load_from decorator for "
+                f"function: {fn.__qualname__}"
+            )
+        if self.inject not in params:
+            raise InvalidDecoratorException(
+                f"Parameter {self.inject} not required by nodes produced by {fn.__qualname__}"
+            )
+        return self.inject
+
+    def inject_nodes(
+        self, params: Dict[str, Type[Type]], config: Dict[str, Any], fn: Callable
+    ) -> Optional[Collection[node.Node]]:
+        pass
```

Comment:
pass?

Reply:
Left in by accident, if you look below it actually ran (`pass` is compiled out/turned into a no-op).